adobe/spectrum-web-components
# docs(dropzone): demonstrate how to announce items added to dropzone

## Description



Added a new storybook example for dropzone to demonstrate the expected behavior of a screen reader when adding items to the drop zone and updated the dropzone's README to communicate the consumer's responsibility for handling this. 

## Related issue(s)



- fixes https://github.com/adobe/spectrum-web-components/issues/3408

## Motivation and context



Existing examples do not capture expected behavior of screen readers when items are dropped into the dropzone

## How has this been tested?




-   [ ] Dragging item into dropzone
    1. Go to dropzone's newly added controlled story
    2. Start VoiceOver by going to accessibility tabs in your settings
    3. Drag item above dropzone into the drop zone
    4. Observe that item is announced
-   [ ] Adding item from desktop into dropzone
    1. Go to dropzone's newly added controlled story
    2. Start VoiceOver by going to accessibility tabs in your settings
    3. Tab to "select a file" link and hit enter
    4. Select some random file (storybook does not read any user input to avoid any security concerns) and exit from fs
    5. Observe that item added is announced 

## Screenshots (if appropriate)

Dragging item into dropzone:

https://github.com/adobe/spectrum-web-components/assets/24359834/15c8b6ad-2b20-4c5c-aadd-28f087ee2e6d

Adding item from personal desktop into drop zone:

https://github.com/adobe/spectrum-web-components/assets/24359834/d11fc42a-1e7c-4945-ab9a-beb2c807c928

## Types of changes




## Checklist




## Best practices

This repository uses conventional commit syntax for each commit message; note that the GitHub UI does not use this by default so be cautious when accepting suggested changes. Avoid the "Update branch" button on the pull request and opt instead for rebasing your branch against `main`.
